src/selectors/DenominationSelectors.js

```diff
@@ -4,7 +4,7 @@ import { type EdgeCurrencyInfo, type EdgeDenomination } from 'edge-core-js'
 
 import type { Dispatch, GetState, RootState } from '../types/reduxTypes.js'
 
-export const emptyEdgeDenomination: EdgeDenomination = {
+const emptyEdgeDenomination: EdgeDenomination = {
```

Comment:
Good catch, exporting a const can lead to trouble.
Also, can you not return the constant directly in the 'getExchangeDenomination' function and instead destruct it into a new object `return {...emptyEdgeDenomination}`.
It's better if we don't share an object between calls.